lib/mavenlink/integrated_resource.rb

```diff
@@ -0,0 +1,24 @@
+module Mavenlink
+  class IntegratedResource < Model
+    def save
+      if valid?
+        reload(create_or_update(attributes_for_create_or_update))
+        true
```

Comment:
Why are we explicitly returning true? What happens if the create or update faills?